# HData.msg
float32[] T
float32[] R
float32[] R_degree